src/renderer/src/components/radio/radio-container.tsx

```diff
@@ -40,11 +39,11 @@ const RadioContainer: React.FC = () => {
   if (!isConnected) {
     return (
       <div className="h-100 mx-3 d-flex justify-content-center flex-column gap-0.5 hide-topbar">
-        <div className="d-flex justify-content-center radio-text  text-center">
-          VATSIM connection found!
+        <div className="d-flex justify-content-center radio-text text-center">
+          VATSIM connection detected!
         </div>
         <div className="d-flex justify-content-center radio-sub-text text-muted text-center">
-          Click the connect button to establish a connection to AFV.
+          Click the connect button to establish a connection to the VATSIM audio network.
```

Comment:
Love the string changes! They're slightly inconsistent though, this one calls it "the VATSIM audio network" and the other one calls it "the VATSIM network". Pick one and use it in both places.

Reply:
the VATSIM network refers to FSD, and the VATSIM audio network refers to AFV

Reply:
Do users know the difference? Does it matter to them?